frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.module.css

```diff
@@ -0,0 +1,25 @@
+.Primary {
+  fill: var(--mb-color-icon-primary);
+}
+
+.SecondaryStroke {
+  stroke: var(--mb-color-icon-secondary);
+}
+
+.SecondaryFill {
+  fill: var(--mb-color-icon-secondary);
+}
+
+.Disabled {
+  & > .Primary {
+    fill: var(--mb-color-icon-disabled);
```

Comment:
if I apply the same pattern for the icon colors. there should be 2 new colors
- `icon-primary-disabled`
- `icon-secondary-disabled`

using `background-disabled` might not make sense since this is an icon. IDK

Reply:
ah I had `primary-disabled` first 😭